atlasdb-cli/build.gradle

```diff
@@ -4,7 +4,7 @@ apply from: '../gradle/shared.gradle'
 
 apply plugin: 'application'
 mainClassName = 'com.palantir.atlasdb.cli.AtlasCli'
-applicationName = 'atlas-cli'
+applicationName = 'atlasdb'
```

Comment:
nit: why have this share a name with the product when it is a distinct cli tool used to manage the product.  I would point to the clean distinction between nodetool and cassandra which makes support discussions much clearer.